main/editor.js

```diff
@@ -19,11 +19,27 @@ let editor = null;
 
 const openEditorWindow = () => {
```

Comment:
Should this take `src` as the filepath to the video that should be edited? @karaggeorge